### ISSUE TITLE
Watch List Not Finding Some Declared Labels

### ISSUE THREAD

**Issue by paulhocker** [Sakrac/IceBroLite#35]
It seems that I cannot declare some of my labels in the watch even though they are defined and show up in the debugger.

![image](https://user-images.githubusercontent.com/2214652/210191595-02f554b0-0c2b-47e5-aaf7-b5af19065e0d.png)

It looks fine when I query it on the debugger.

![image](https://user-images.githubusercontent.com/2214652/210191740-886e9c20-5759-4fa2-8424-a990a03f0885.png)

**Comment by Sakrac** [comment]
It is hard to debug something I can't repro, but can you try to drag the symbol to the watch window title and see if that gives a different result? It would help narrow it down a bit.

**Comment by paulhocker** [comment]
Video below.

I did a bit more testing and I renamed the Label to IND_PL and guess what, it works now? So, it would seem that is has something to do with the label starting with one character followed by a underscore? Seems silly. :)


https://user-images.githubusercontent.com/2214652/210294312-0e762bb8-6c02-45f3-baab-b6781e94298a.mp4

![image](https://user-images.githubusercontent.com/2214652/210294355-510754a7-948d-4054-b021-0bc55e0b4a90.png)
